src/Core/Services/Implementations/OrganizationService.cs

```diff
@@ -1135,44 +1135,34 @@ public async Task<OrganizationUser> AcceptUserAsync(string orgIdentifier, User u
                 }
             }
 
-            var existingOrgUserCount = await _organizationUserRepository.GetCountByOrganizationAsync(
-                orgUser.OrganizationId, user.Email, true);
-            if (existingOrgUserCount > 0)
-            {
-                throw new BadRequestException("You are already part of this organization.");
-            }
-
-            if (!CoreHelpers.UserInviteTokenIsValid(_dataProtector, token, user.Email, orgUser.Id, _globalSettings))
```

Comment:
Was this comment looking at a specific commit? This logic was moved around on [this PR](https://github.com/bitwarden/server/pull/945/files), it's reflected in some commits here because of merge issues.